runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -358,6 +359,11 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 		{
 			ObjectComposition object = plugin.getClient().getObjectDefinition(getObjectId());
 
+			if (Strings.isNullOrEmpty(object.getName()) || "null".equals(object.getName()))
+			{
+				object = object.getImpostor();
```

Comment:
Line 361 implies object can be null. Also do not call getImpostor without first checking impostorids is not null.